build and push docker on release (#213)
reef-technologies/B2_Command_Line_Tool: README.md

### Docker image

#### Authorization

> User can either authorize on each call (`list-buckets` is just an exemplary command here)

Reviewer: ```suggestion
User can either authorize on each command (`list-buckets` is just a example here)
```
